src/averageAges.js

```diff
@@ -37,7 +42,16 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const womens = withChildren
+    ? people.filter(woman =>
+      (people.find(child => woman.name === child.mother)))
```

Comment:
use people.some(...) instead. find is used usually when you need to find an item and perform some actions with it. In this case you just need to check that there is such person with specified name